### PR TITLE
release(v1.9.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.9.0-alpha.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,301 @@
+## [Talos 1.9.0-alpha.0](https://github.com/siderolabs/talos/releases/tag/v1.9.0-alpha.0) (2024-10-18)
+
+Welcome to the v1.9.0-alpha.0 release of Talos!  
+*This is a pre-release of Talos*
+
+Starting with Talos v1.8.0, only standard assets would be published as github release assets. These include:
```
Comment (frezbo) [inline CHANGELOG.md:6]: this should have been dropped

**Review by frezbo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:6]: yep, thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,301 @@
...
+The `talosctl cgroups` command has been added to the `talosctl` tool.
+This command allows you to view the cgroup resource consumption and limits for a machine, e.g.
+`talosctl cgroups --preset memory`.
+
+
+### Component Updates
+
+Linux: 6.6.57
+containerd: 2.0.0-rc.5
+Flannel: 0.25.7
```
Comment (dsseng) [inline CHANGELOG.md:31]:
```suggestion
Linux: 6.6.57
containerd: 2.0.0-rc.5
Flannel: 0.25.7
systemd-udevd: 256.6 replaced eudev
```

**Review by dsseng — APPROVED** [review]
Otherwise good

File: CHANGELOG.md (hunk 2 above)
Comment (smira) [reply CHANGELOG.md:31]: this should have been added in the PR replacing eudev, but I will add a note here ;)

Comment (dsseng) [reply CHANGELOG.md:31]: thanks!

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**